libs/sdk-ui-dashboard/src/presentation/automationFilters/useAutomationDashboardFilters.ts

```diff
@@ -23,18 +23,22 @@ export const useAutomationDashboardFilters = ({
     automationFilters: FilterContextItem[] | undefined;
     allVisibleFiltersMetadata?: IAutomationVisibleFilter[] | undefined;
 }): IUseAutomationDashboardFiltersResult => {
+    // Is dashboard stored with "frozen" filters or is it using the latest saved ones?
     const areFiltersStored = useMemo(
         () =>
             editAutomation?.exportDefinitions?.some((exportDefinition) => {
-                return (exportDefinition.requestPayload.content.filters?.length ?? 0) > 0;
+                return (exportDefinition.requestPayload.content.filters?.length ?? 0) > 0; // is this correct? 0 length should be no filters??
```

Comment:
This was ok while we were storing even the common date filter with all time value as it was not possible to remove it. Now this may be a problem as the user may remove every filter and all time common will no be saved. This is really an edge case, but I don't think we have any other way to check this